spec/rest-api/ddl-service.yaml

```diff
@@ -0,0 +1,500 @@
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/generalResp"
+    delete:
+      summary: Drop an existed tabular table named table_name
+      parameters:
+        - name: table_name
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: x-bdg-request-id
+          in: query
+          required: false
+          description: the unique id to identify a ddl request
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/generalResp"
+
+  /rawstores:
+    post:
+      summary: Create a rawstore table
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/rawstoreTable"
+      parameters:
+        - name: x-bdg-request-id
+          in: query
+          required: false
+          description: the unique id to identify a ddl request
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/generalResp"
+    get:
+      summary: Get list of rawstore tables
+      parameters:
+        - name: x-bdg-request-id
+          in: query
+          required: false
+          description: the unique id to identify a ddl request
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/rawstoreTableListResp"
+
+  /rawstores/{table_name}:
+    patch:
+      summary: Update an existed rawstore table named table_name
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/updateTable"
+      parameters:
+        - name: table_name
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: x-bdg-request-id
+          in: query
+          required: false
+          description: the unique id to identify a ddl request
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/generalResp"
+    delete:
+      summary: Drop an existed rawstore table named table_name
+      parameters:
+        - name: table_name
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: x-bdg-request-id
+          in: query
```

Comment:
Is this in `query` or in `head` or both ?

Reply:
It is in query.

Reply:
It is in query.

Reply:
x-bdg-request-id is in query while X-ClickHouse-Query-Id is in header

Reply:
`x-bdg-request-id` should be in header as well.

Reply:
reset it into header